api/command_test.go

```diff
@@ -184,3 +185,53 @@ func TestEchoCommand(t *testing.T) {
 		t.Fatal("Echo command failed to send")
 	}
 }
+
+func TestLoadTestUrlCommand(t *testing.T) {
+	Setup()
+
+	// enable testing to use /loadtest but don't save it since we don't want to overwrite config.json
+	utils.Cfg.ServiceSettings.EnableTesting = true
```

Comment:
Oh that's right. I assumed Setup() reloaded the config every time it ran, but I just looked at the code for it and it actually doesn't do that.